runtime/altair/Cargo.toml

```diff
@@ -459,7 +460,7 @@ try-runtime = [
 # generate the metadata hash and then a second time with the
 # `RUNTIME_METADATA_HASH` environment variable set for the `CheckMetadataHash`
 # extension.
-metadata-hash = ["substrate-wasm-builder/metadata-hash"]
+metadata-hash = ["substrate-wasm-builder?/metadata-hash"]
```

Comment:
Why the `?`?. is not your intention to always set the `metadata-hash` of `substrate-wasm-builder` each time you set `metadata-hash`